### DISCUSSION TITLE
Upon testing, we have noticed a few issues with the generations, please have a look:

### DISCUSSION THREAD

**Discussion by oldhand7** [lllyasviel/Fooocus#2906] [Q&A] [answered]
- Upon testing, we have noticed a few issues with the generations, please have a look.

![95c9d926-ff39-44a6-b90e-79560eabdef4](https://github.com/lllyasviel/Fooocus/assets/91205658/352826ac-0b46-4be8-bdf8-13dcee4c1329)
![9218bbcf-e38e-495f-88cb-0455def67132](https://github.com/lllyasviel/Fooocus/assets/91205658/d92cceab-1ffa-4c93-b03b-edf4227e5df8)
![e2c04cbc-4a63-4a1d-8002-df0ff79f050c](https://github.com/lllyasviel/Fooocus/assets/91205658/43eb2a75-3cce-41ab-9d0b-228bc14ac516)
![0def22a2-7085-4f70-b760-99544c8d6551](https://github.com/lllyasviel/Fooocus/assets/91205658/9ba04b00-5536-4a48-b6b7-c3ae23ab488d)

- And asian females are not generated well.

Please let me know what I can do for better ouput. Thank you.

**Comment by mashb1t** [comment, marked as answer]
1000x1400 is not a native resolution of SDXL (has not been trained on this).
=> Choose another native resolution to generate non-repeating content, see https://github.com/lllyasviel/Fooocus/pull/1617#issuecomment-1870469961

**Comment by oldhand7** [comment]
Thank you for your help. 
I 've noticed that asian are not generated well.
In this image, the eye is distorted. Please let me know what I can do for better result. Thanks.
![image_720](https://github.com/lllyasviel/Fooocus/assets/91205658/e12a3864-0527-4689-8d3a-020fa26c2451)

**Reply by mashb1t** [reply]
upscale / inpaint / better prompt can help

**Comment by 841372** [comment]
Send me the girl, I will make her give birth
